docs/conventions.rst

```diff
@@ -0,0 +1,151 @@
+(as well as the page on `Nasmyth telescopes <https://en.wikipedia.org/wiki/Nasmyth_telescope>`_).
+We limit our focus here to those mount types presently supported in pyuvdata, which
+include:
+
+  - ``"alt-az"``: Arguably the most common form of antenna mount, where the two axes of
+    rotation rotate in azimuth and elevation (i.e., "altitude"), and "up" from the
+    perspective of the receiver points toward zenith. For a mount of this type, a feed
+    angle of 0 degrees is aligned with the parallactic angle at the given position of
+    the sky.
+  - ``"equatorial"``: Similar to "alt-az", but where one axis of rotation is aligned
+    to the celestial equatorial plane (i.e., where the declination is zero), such that
+    the two axes of rotation are aligned to the right ascension/hour angle and
+    declination axes. For a mount of this type, a feed angle of 0 degrees is aligned
+    with north on the (apparent) celestial sphere.
+  - ``"orbiting"``: Denotes that the antenna is in orbit, such that the orientation of
+    the antenna changes with time. This mode appears to have been added to UVFITS to
+    support the `VLBI Space Observatory Programme <https://en.wikipedia.org/wiki/HALCA>`_.
+    pyuvdata does not currently support additional orbital parameters.
+  - ``"x-y"``: A mount type that is somewhat prevalent for tracking of LEO and MEO
+    satellites due to its ability to `track more easily through zenith than "alt-az"
+    mounts <https://ntrs.nasa.gov/api/citations/19650021134/downloads/19650021134.pdf>`_,
+    this has one rotation axis aligned to local north-south, and the other lies in the
+    plane of the great circle connecting local east-west through zenith. For this mount
+    type, a feed angle of 0 degrees varies with sky position, aligned to
+    :math:`\arctan(\cos(\textrm{HA}) / (\sin(\textrm{HA}) * \sin(\delta)))`,
+    where :math:`\delta` is the declination of the source and HA the hour angle.
+  - ``"alt-az+nasmyth-r"``: Similar to "alt-az", with the addition of a flat tertiary
+    mirror that allows the detector to sit at a fixed elevation while the antenna primary
+    moves up and down in elevation. For this "right-handed" Nasmyth variant, a feed angle
+    of 0 degrees will be aligned to sum of the parallactic and elevation angles.
+  - ``"alt-az+nasmyth-l"``: The "left-handed" variant of the Nasmyth mount, where a
+    feed angle of 0 degrees is aigned to the **difference** of the parallactic and
+    elevation angles.
+  - ``"phased"``: Denotes an instrument where the an individual antenna input is
+    a phased array of detectors that are "beamformed" into a single voltage stream
+    (otherwise sometimes referred to as "electronically steered"). For this mount, a
+    0-degree feed angle is aligned to the great circle that connects local north,
+    local south, and the azimuth and elevation position of the phased antenna beam. Note
```

Comment:
This feels wrong to me because it would imply that we had to write different feed angles into files depending on the beamformer settings. I think if the physical antennas do not move (e.g. MWA, SKA-low) that the feed angles should always be the same and should be recorded in the same way as they are for fixed antennas, so 0 should be north and 90 should be east.

Reply:
I think you're right -- I managed to convince myself of a subtlety that in hindsight doesn't actually exist, and 0 degrees should be parallel w/ local north. I'll fix that here shortly...